### PR TITLE
Add new page(s) to the wiki

### PR DESCRIPTION
**Pull request by svr333**
I'll try to add at least one page each day during this week.
I'd love for someone to check over / edit / comment on these if anything is wrong/incomplete.

## New pages
- Hazmat suit

### CODE REVIEW

File: pages/Hazmat-Suit.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+| Hazmat Suit  |  Fire Resistance  |
+| Hazmat Suit Leggings |  N/A  |
+| Rubber Boots |  N/A  |
+
+## Obtaining
+Each piece can be crafted in the [Armor Forge](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Armor-Forge).
+
+## Usage
+This set is mostly used to provide protection against [Radiation](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Radiation).<br>
+The armour has poor durability, therefor it's best to only use it when necesarry.
```
Comment (TheBusyBiscuit) [inline pages/Hazmat-Suit.md]: *therefore

Comment (TheBusyBiscuit) [inline pages/Hazmat-Suit.md]: *necessary

File: pages/Hazmat-Suit.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Hazmat suit is part of the [Armor](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Armor) category.<br>
```
Comment (TheBusyBiscuit) [inline pages/Hazmat-Suit.md]: *The Hazmat Suit

File: pages/Hazmat-Suit.md (hunk 1 above)
Comment (svr333) [reply pages/Hazmat-Suit.md]: I can English

File: pages/Armor-Forge.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+The Armor Forge is in the [Basic Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machines) category. It is a multiblock structure.<br>
+
+## Obtaining
+The Armor Forge is a multiblock structure, it consists of a dispenser and an anvil.<br>
+**Note:** The dispenser has to be facing upwards and the anvil cannot be (slightly) broken.<br>
```
Comment (TheBusyBiscuit) [inline pages/Armor-Forge.md]: "cannot be broken" sounds like you are not able to break the block.
I think you should rephrase it to:
"Note that it must be a standard anvil, damaged Anvils will not work".
And the fact about the dispenser is wrong, it doesn't have to be facing that way, it just looks nicer.

File: pages/Hazmat-Suit.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+| Hazmat Suit  |  Fire Resistance  |
+| Hazmat Suit Leggings |  N/A  |
+| Rubber Boots |  N/A  |
+
+## Obtaining
+Each piece can be crafted in the [Armor Forge](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Armor-Forge).
+
+## Usage
+This set is mostly used to provide protection against [Radiation](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Radiation).<br>
+The armour has poor durability, therefore it's best to only use it when necessary.
```
Comment (TheBusyBiscuit) [inline pages/Hazmat-Suit.md]: *armor

(I know, I know, but let's be consistent on this page after writing it without an u for 2 times already...)

File: pages/Armor-Forge.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+A complete setup should look like this:
+
+![Armor Forge Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-armor-forge.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the Dispenser to collect your crafted armor.
```
Comment (TheBusyBiscuit) [inline pages/Armor-Forge.md]: Maybe add that it will be placed into the dispenser otherwise.

File: pages/Hazmat-Suit.md (hunk 4 above)
Comment (svr333) [reply pages/Hazmat-Suit.md]: Sorry, it's a force of habit to write it with a u, I went over it to remove the u's but I've skipped over this one :)

File: pages/Ore-Washer.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+## Obtaining
+The Ore Washer is a multiblock structure, it consists of a cauldron, a fence and a dispenser.<br>
+**Note:** It doesn't matter which side the dispenser is facing, place it to your liking.<br>
+
+A complete setup should look like this:
+
+![Armor Forge Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-ore-washer.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the Dispenser to collect your crafted armor.
```
Comment (TheBusyBiscuit) [inline pages/Ore-Washer.md]: The Ore Washer does not create armor...

File: pages/Armor-Forge.md (hunk 3 above)
Comment (svr333) [reply pages/Armor-Forge.md]: I thought it had to be facing that way, since it says so in the sf guide

Comment (svr333) [reply pages/Armor-Forge.md]: But I'll definitely rephrase

File: pages/Armor-Forge.md (hunk 5 above)
Comment (svr333) [reply pages/Armor-Forge.md]: 👍

File: pages/Ore-Washer.md (hunk 6 above)
Comment (svr333) [reply pages/Ore-Washer.md]: Yeah I fixed that as soon as I pushed it lol

File: pages/Hazmat-Suit.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+| Scuba Helmet  |  Water breathing  |
+| Hazmat Suit  |  Fire Resistance  |
+| Hazmat Suit Leggings |  N/A  |
+| Rubber Boots |  N/A  |
+
+## Obtaining
+Each piece can be crafted in the [Armor Forge](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Armor-Forge).
+
+## Usage
+This set is mostly used to provide protection against [Radiation](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Radiation).<br>
```
Comment (TheBusyBiscuit) [inline pages/Hazmat-Suit.md]: You should probably add that you need to wear the full set in order for this to happen :?

File: pages/Ore-Washer.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+**Note:** It doesn't matter which side the dispenser is facing, place it to your liking.<br>
+
+A complete setup should look like this:
+
+![Armor Forge Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-ore-washer.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your washed ores. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Ore Washer can be used to wash ores. Most common use is to wash [Sifted Ore](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Sifted-Ore) into [Dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts).<br>
```
Comment (TheBusyBiscuit) [inline pages/Ore-Washer.md]: "wash ores" sounds misleading, as you cannot place ores into this machine. Maybe remove that part and only say that it is mostly used for sifted ore -> dusts.

Comment (svr333) [reply pages/Ore-Washer.md]: Sure

File: pages/Hazmat-Suit.md (hunk 7 above)
Comment (svr333) [reply pages/Hazmat-Suit.md]: Yup I'll mention that

**Comment by TheBusyBiscuit** [comment]
Alright, this PR is good to go.
One more thing though:

The Slimefun Guide shows a link to the wiki page when viewing an Item, this is done via a json file.
It would be awesome if you could add any new pages you add to that file too.
Here is a link:
https://github.com/TheBusyBiscuit/Slimefun4/blob/master/src/main/resources/wiki.json
